src/test/kotlin/io/github/nomisrev/routes/TagRouteSpec.kt

```diff
@@ -34,8 +30,8 @@ class TagRouteSpec :
       withServer {
         val response = get(TagsResource()) { contentType(ContentType.Application.Json) }
 
-        response.status shouldBe HttpStatusCode.OK
-        response.body<TagsResponse>().tags.toSet() shouldBe emptySet()
+        assert(response.status == HttpStatusCode.OK)
+        assert(response.body<TagsResponse>().tags.isEmpty())
```

Comment:
I think here we need to use `== emptyList()` in order to get a proper error message from power-assert.

```suggestion
        assert(response.body<TagsResponse>().tags == emptyList())
```

Reply:
I think the message for `isEmpty` is good enough, but if you prefer an equality check, I will refactor it. Here is a comparison:
![image](https://github.com/nomisRev/ktor-arrow-example/assets/6090010/b4558fc4-0a8c-49c4-a3f7-c5f1fc9bdff7)

![image](https://github.com/nomisRev/ktor-arrow-example/assets/6090010/18fb99b5-67b3-4263-88d0-6be26a8db5b5)